controller/canphone/hwcontrols.py

```diff
@@ -1,7 +1,34 @@
-import gpiozero
+import thread
+from gpiozero import LED, Button
+from signal import pause
+from time import sleep
 
-# implement controller where it is possible to
-# - control leds and beeper for incoming calls
-# - read status of the hangup switch
-# - register callback for getting the hangup status of the can
-# - register callback for Push2Talk button
```

Comment:
If this is a class you can add functions to register callbacks. Have a look at the MQTT code if you need some inspiration how this could look like.
I also have commented out invocations of the callback in the controller class.